data/FilterList.json

```diff
@@ -9745,191 +9745,191 @@
   },
   {
     "id": 481,
-    "chatUrl": null,
-    "description": null,
-    "descriptionSourceUrl": null,
+    "chatUrl": "https://t.me/EnergizedProtection",
+    "description": "True Lite Hosts from Energized. Only 1MB in Size.",
+    "descriptionSourceUrl": "https://github.com/EnergizedProtection/block",
     "discontinuedDate": null,
-    "donateUrl": null,
+    "donateUrl": "shadmansaleh@codeian.com",
     "emailAddress": "mail.energized@protonmail.com",
-    "forumUrl": null,
-    "homeUrl": "https://ador.chorompotro.com/energized",
-    "issuesUrl": "https://github.com/EnergizedProtection/EnergizedHosts/issues",
-    "licenseId": 2,
-    "name": "Energized Ad Protection",
-    "policyUrl": null,
+    "forumUrl": "https://forum.xda-developers.com/android/software-hacking/energized-protection-ad-porn-malware-t3803773",
+    "homeUrl": "https://nayemador.com/energized",
+    "issuesUrl": "https://github.com/EnergizedProtection/block/issues",
+    "licenseId": 9,
+    "name": "Energized Spark Protection",
+    "policyUrl": "https://nayemador.com/energized/license",
```

Comment:
`policyUrl` should be more a page about what types of rules are acceptable in the list. `licenceId` handles licensing.